core/src/main/java/org/frankframework/align/Json2Xml.java

```diff
@@ -91,7 +135,14 @@ public void startParse(JsonValue node) throws SAXException {
 		if (node instanceof JsonArray && !insertElementContainerElements && strictSyntax) {
 			throw new SAXException(MSG_EXPECTED_SINGLE_ELEMENT+" ["+getRootElement()+"] or array element container");
 		}
-		super.startParse(node);
+		//if (log.isTraceEnabled()) log.trace("startParse() rootNode ["+node.toString()+"]"); // result of node.toString() is confusing. Do not log this.
```

Comment:
Kan dit comment weg?

Reply:
Ik twijfelde erover en heb hem toch maar laten staan. Maar in de toekomst kan die misschien alsnog wel weg.